### PR TITLE
CHANGELOG.md: Add v1.3.0-rc.0 entry

### PR DESCRIPTION
**Pull request by brancz**
We've accumulated a number of features over the past month, and since we've traditionally cut releases right around Kubernetes releases, here it goes.

I encourage everyone to test this so we can find bugs, should there be any new ones introduced.

@andyxning

### CODE REVIEW

**Comment by brancz** [comment]
@piosz @loburm please stand by for pushing the images to gcr.io :slightly_smiling_face:

**Comment by andyxning** [comment]
@brancz Do we need to wait before #402 addressed?

**Comment by brancz** [comment]
I'm more than happy to do a patch release if we do figure out what's happening. Right now the issue seems like we are having a problem to debug/find the problem in the first place.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+* [FEATURE] Add `kube_node_spec_taint` metric.
+* [FEATURE] Add `kube_namespace_annotations` metric.
+* [FEATURE] Add `kube_deployment_spec_strategy_rollingupdate_max_surge` metric.
+* [FEATURE] Add `kube_persistentvolume_labels` metric.
+* [FEATURE] Add `kube_persistentvolumeclaim_lables` metric.
+* [FEATURE] Add `kube_daemonset_labels` metric.
+* [ENHANCEMENT] Add additional reasons to `kube_pod_container_status_waiting_reason` metric.
 * [BUGFIX] Fix namespacing of HPA.
+* [BUGFIX] Fix namespacing of PersistentVolumes.
+* [BUGFIX] Fix CronJob tab parsing.
```
Comment (andyxning) [inline CHANGELOG.md]: Can you wait for this about two days. I still have some confusion about cronjob next schedule time parsing. Will confirm this in about two days.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,17 @@
-## Unreleased
+## v1.3.0-rc.0 / 2018-03-23
 
-* [FEATURE] Add `kube_pod_completion_time` metric.
```
Comment (andyxning) [inline CHANGELOG.md]: Another important feature:
* add secret and configmap metrics. #379

File: CHANGELOG.md (hunk 1 above)
Comment (brancz) [reply CHANGELOG.md]: Ack, no problem.

File: CHANGELOG.md (hunk 2 above)
Comment (brancz) [reply CHANGELOG.md]: Thanks! Added.

**Comment by andyxning** [comment]
Once https://github.com/kubernetes/kube-state-metrics/issues/402 is merged. It should be fine to merge this PR and cut the release. :)

**Comment by brancz** [comment]
Container image built and pushed to `quay.io/coreos/kube-state-metrics:v1.3.0-rc.0`.

**Comment by loburm** [comment]
I have also built and pushed images:
`gcr.io/google_containers/kube-state-metrics:v1.3.0-rc.0`
`gcr.io/google_containers/kube-state-metrics-amd64:v1.3.0-rc.0`

**Pull request merged**